modules/mobile/app/models/mobile/v0/contracts/claims_and_appeals.rb

```diff
@@ -7,6 +7,7 @@ class ClaimsAndAppeals < PaginationBase
         params(Schemas::DateRangeSchema) do
           optional(:use_cache).maybe(:bool, :filled?)
           optional(:show_completed).maybe(:bool, :filled?)
+          optional(:decision_letter_sent).maybe(:bool, :filled?)
```

Comment:
Is this necessary? These contracts are for validating incoming params from the mobile client? Unless we're planning to filter based on this value, I doubt this is a controller param.

Reply:
oops. yeah that slipped in when I was trying to debug something and forgot to remove it. fixed